src/Controls/tests/TestCases.HostApp/Issues/Issue11655.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue11655">
+
+  <VerticalStackLayout Spacing="25"
+                       VerticalOptions="Center">
+    <Label x:Name="label"
```

Comment:
Could update the sample including another Label using RTL?

Reply:
I have modified the test case.